packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/react/load-app.md

```diff
@@ -0,0 +1,68 @@
+### Set up the Okta configuration settings
+
+Use the required [configuration settings](#okta-org-app-integration-configuration-settings) to initialize your Sign-In Widget and your Auth JS instance:
+
+* `clientId`: Your client ID &mdash; `${yourClientId}`
+* ` issuer`: The authorization server in your Okta org (for example, `https://${yourOktaDomain}/oauth2/default`)
+* `useInteractionCodeFlow`: Set this option to `true` to enable the [Interaction Code flow](/docs/concepts/interaction-code/#the-interaction-code-flow) in the embedded Widget.
+* `scopes`: Set the OAuth 2.0 scopes that your app requires. For example, `['openid', 'profile', 'email']` are commonly used scopes. See [Scopes](/docs/reference/api/oidc/#scopes) for details on additional supported scopes.
+* `redirectUri`: Set your callback redirect URI. This value must be configured in your Okta app **Sign-in redirect URIs** and **Trusted Origins** lists.
+
+You can create a `src/config.js` file to define your configuration settings. For example:
+
+```js
+export default {
+  oidc: {
+    issuer: 'https://${yourOktaDomain}/oauth2/default',
+    clientId: '${clientId}',
+    scopes: ['openid', 'profile', 'email'],
+    redirectUri: `${window.location.origin}/login/callback`,
+    pkce: true
+  },
+  widget: {
+    issuer: 'https://${yourOktaDomain}/oauth2/default',
+    clientId: '${clientId}',
+    redirectUri: `${window.location.origin}/login/callback`,
+    scopes: ['openid', 'profile', 'email'],
+    useInteractionCodeFlow: true
+  },
+};
+```
```

Comment:
I get a warning from ESLint in IntelliJ IDEA with this file.

<img width="835" alt="Screen Shot 2022-02-23 at 09 06 15" src="https://user-images.githubusercontent.com/17892/155358608-86975b88-b39c-4056-81e0-a90e0a24a610.png">